hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/upgrade/TestAbstractLayoutVersionManager.java

```diff
@@ -80,6 +81,7 @@ public void testInitializationWithUpToDateMetadataVersion() throws Exception {
 
     assertFalse(versionManager.needsFinalization());
     assertFalse(versionManager.unfinalizedFeatures().iterator().hasNext());
+    versionManager.close();
```

Comment:
shouldn't these close in finally block, else if the test fails post init, it won't cleanup, right?

Reply:
Yes, you're right.

Added the close call to junit `AfterEach` so that it will be executed after every test execution.